src/input.service.ts

```diff
@@ -475,7 +480,7 @@ export class InputService {
    * a connected gamepad somewhere.
    */
   private pollGamepad(now: number) {
-    const rawpads = navigator.getGamepads().filter(pad => !!pad); // refreshes all checked-out gamepads
+    const rawpads = Array.from(navigator.getGamepads()).filter(pad => !!pad); // refreshes all checked-out gamepads
```

Comment:
That's why we shim 😄 

![](https://peet.io/i/17-01-8s4jllwr4th7das6zot0room90p6aa.png)